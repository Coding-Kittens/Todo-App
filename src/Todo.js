const Todo  = ({todoText,id,deleteTodo})=>{

return <div className='Todo'>
  <p>{todoText}</p>
  <button className='Todo_button' id={id} type="button" onClick={deleteTodo} >X</button>
  </div>
}

export default Todo ;
