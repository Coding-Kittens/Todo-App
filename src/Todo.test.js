import {render} from '@testing-library/react';
import Todo from './Todo';

it('should renders without crashing',()=>{
  render(<Todo/>);
})

it('should match snapshot',()=>{
const {asFragment} = render(<Todo/>);
expect(asFragment()).toMatchSnapshot();
})
