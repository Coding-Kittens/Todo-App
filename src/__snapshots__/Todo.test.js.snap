// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Todo"
  >
    <p />
    <button
      class="Todo_button"
      type="button"
    >
      X
    </button>
  </div>
</DocumentFragment>
`;
